src/login/logind-user.c

```diff
@@ -525,9 +525,7 @@ static int user_stop_service(User *u) {
                 return r;
         }
 
-        free(u->service_job);
-        u->service_job = job;
-
+        free_and_replace(u->service_job, job);
```

Comment:
hmm, coccinelle should have found that, but it doesn't. something is weird with coccinelle